website/content/en/docs/troubleshooting.md

```diff
@@ -37,6 +37,17 @@ helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter \
   ...
 ```
 
+#### Check AWS resources with Karpenter Discovery tag
```

Comment:
I'm skeptical of how useful it is to get all the resources across the account that have a given tag/key combination. Does it make more sense to do specific calls to the `describe-subnets` and `describe-security-groups` endpoints?